rocAL/rocAL/include/master_graph.h

```diff
@@ -79,7 +79,7 @@ class MasterGraph
     Image *create_loader_output_image(const ImageInfo &info);
     MetaDataBatch *create_label_reader(const char *source_path, MetaDataReaderType reader_type);
     MetaDataBatch *create_video_label_reader(const char *source_path, MetaDataReaderType reader_type, unsigned sequence_length, unsigned frame_step, unsigned frame_stride, bool file_list_frame_num = true);
-    MetaDataBatch *create_coco_meta_data_reader(const char *source_path, bool is_output);
+    MetaDataBatch *create_coco_meta_data_reader(const char *source_path, bool is_output, MetaDataReaderType reader_type , MetaDataType label_type, bool keypoint, float sigma = 0.0, unsigned pose_output_width = 0, unsigned pose_output_height = 0);
```

Comment:
Why do we have a bool keypoint variable when we can deciper that from the reader type and label type?